sunpy/map/sources/hinode.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, data, header, **kwargs):
         self.meta['telescop'] = "Hinode"
         self.meta['wavelnth'] = np.nan
         self.meta['waveunit'] = 'keV'
-        self._nickname = "{0} {1}-{2}".format(self.detector, fw1, fw2)
+        self._nickname = "XRT"
```

Comment:
Shouldn't be the nikname the whole: XRT and detector? or you just want to show XRT without any other information?
Does this affect when plotting a map (as the title of the plot)?